src/Pages/Login.js

```diff
@@ -28,12 +28,15 @@ export default function Login() {
             });
 
             const data = await response.json();
-           
+
             if (data.accessToken) {
                 localStorage.setItem("accessToken", data.accessToken);
                 resetForm();
                 navigate("/");
-            } 
+            }
+            else{
+                alert("Abey Yaaar!! Dekh kr password daal")
+            }
```

Comment:
Abe yrr! dekh alert daal

Reply:
Invalid k liye vo notification vale bht sare daal kr dekhe h ni chl rhe pr koi b

Reply:
To last me ye dalla h

Reply:
Hta du ky

Reply:
For invalid you have throw error. not set any alert.